app/stac_api/management/commands/calculate_extent.py

```diff
@@ -0,0 +1,94 @@
+from stac_api.utils import CommandHandler
+
+
+def boolean_input(question, default=None):
+    result = input(f"{question}")
+    if not result and default is not None:
+        return default
+    return len(result) > 0 and result[0].lower() == "y"
+
+
+class Handler(CommandHandler):
+
+    def run(self):
+        self.print_success('Runing command to update collection extents')
+        # print(self.options)
+        qry = Collection.objects.filter(extent_out_of_sync=True)
+        if self.options['all']:
+            qry = Collection.objects
+        collections = qry.values_list('id', flat=True)
+
+        # Prompt user to confirm update of all collections if force was not provided.
+        if self.options['all'] and not self.options['force']:
+            cont = boolean_input(
+                f"You are about to update {len(collections)} collections!\n" +
+                "Are you sure you want to continue? (y/n): ",
+                False
+            )
+            if not cont:
+                self.print_warning("Aborted")
+                self.print_success("No collections updated")
+                return
+
+        start = time.monotonic()
+        with connection.cursor() as cursor:
+            for collection_id in collections:
+                cursor.execute(
+                    """
+                    -- Compute collection extent
+                    WITH collection_extent AS (
+                        SELECT
+                            item.collection_id,
+                            ST_SetSRID(ST_EXTENT(item.geometry),4326) as extent_geometry,
+                            MIN(LEAST(item.properties_datetime, item.properties_start_datetime))
+                                as extent_start_datetime,
+                            MAX(GREATEST(item.properties_datetime, item.properties_end_datetime))
+                                as extent_end_datetime
+                        FROM stac_api_item AS item
+                        WHERE item.collection_id = %s
+                        GROUP BY item.collection_id
+                    UNION
+                        -- This covers the case that the last item of a collection is deleted.
+                        SELECT %s AS collection_id, NULL, NULL, NULL
+                    LIMIT 1
+                    )
+                    -- Update related collection extent
+                    UPDATE stac_api_collection SET
+                        extent_out_of_sync = FALSE,
+                        extent_geometry = collection_extent.extent_geometry,
+                        extent_start_datetime = collection_extent.extent_start_datetime,
+                        extent_end_datetime = collection_extent.extent_end_datetime
+                    FROM collection_extent
+                    WHERE id = collection_extent.collection_id;
+                    """, [collection_id, collection_id]
+                )
+                self.print_success(f"collection.id={collection_id} extent updated.")
```

Comment:
`print_success` writes to `stdout`, which makes sense since these commands are mostly executed in an interactive shell. In case they are executed as cron job we need to have the output in a log message to ensure proper json-formatting (otherwise it cannot be correctly parsed by the log pipeline)